filters/numeric/README.md

```diff
@@ -0,0 +1,21 @@
+## Numeric filter
+
+## What type of filter is this?
+
+This filter filters example which contain a numeric value.
+Author: Suchitra Dubey
+Email: suchitra27288@gmail.com
+Affiliation: Acko
+
+## Why is measuring performance on this split important?
+This filter can be used to create splits. Filtering out and testing on examples 
+contain numeric values can provide feedback for improving training data 
+accordingly.
+
+python evaluate.py -f TextContainsNumericFilter -p=1
+
```

Comment:
You will need to run this on your PC.

Reply:
Working on it.

Reply:
![numeric_filter_eval_results](https://user-images.githubusercontent.com/86559954/131537304-e3aa83db-4bf7-4738-a4e9-49bd5a513ce6.png)
Added two evaluation results, one with SNLI and another one with MNLI.

Reply:
Thanks @SuchitraDubey for sharing the results.